Runtime/safe-ds/safe_ds/plotting/_plot_correlation_heatmap.py

```diff
@@ -23,6 +23,6 @@ def plot_correlation_heatmap(table: Table) -> None:
         if not column.type.is_numeric():
             raise NonNumericColumnError(column.name)
     # noinspection PyProtectedMember
-    sns.heatmap(data=table._data.corr(), vmin=-1, vmax=1)
+    ax = sns.heatmap(data=table._data.corr(), vmin=-1, vmax=1, xticklabels=table.get_column_names(), yticklabels=table.get_column_names())
```

Comment:
```suggestion
    sns.heatmap(data=table._data.corr(), vmin=-1, vmax=1, xticklabels=table.get_column_names(), yticklabels=table.get_column_names())
```

Should satisfy linter